src/components/Card.js

```diff
@@ -17,42 +33,92 @@ class Card {
     this._cardImageEl = this._cardElement.querySelector(".card__image-card");
     this._cardImageEl.src = this._link;
     this._cardImageEl.alt = this._title;
+    this._cardLikesEl = this._cardElement.querySelector(".card__button-count");
+    this._cardLikesEl.textContent = this._getLikes();
+    this._trashEl = this._cardElement.querySelector(".card__button-del");
+    if (this._cardOwnerId !== this._myId) {
+      this._trashEl.classList.add("card__button-del-inactive");
+    }
+
+    if (this.isLiked()) {
+      this._cardElement
+        .querySelector(".card__button-like")
```

Comment:
**Could be improved**

All elements that are found every time you call a method should be defined as **class fields** in the constructor, `_setEventListeners` (or in `generateCard`) only **1 time** not to waste resources on searching them again and again.

`Class fields` (the so called **class variables** with `this`) are accessible inside any method of the class. You shouldn’t pass **class fields** into the call of any method because you can use class fields inside all methods.
There can be `this._inputs`, `this._submitButton`, `this._cardImage`, `this._form` , `this._likeButton` and so on.

Reply:
Moved all variables in getCardElement to class fields. Retains functionality, no errors